async-request-reply/src/AsyncOperationStatusChecker.cs

```diff
@@ -76,22 +77,22 @@ public static class AsyncOperationStatusChecker
             }
         }
 
-        private static async Task<IActionResult> OnCompleted(OnCompleteEnum OnComplete, CloudBlockBlob inputBlob, string thisGUID)
+        private static async Task<IActionResult> OnCompleted(OnCompleteEnum OnComplete, BlockBlobClient inputBlob, string thisGUID)
         {
             switch (OnComplete)
             {
                 case OnCompleteEnum.Redirect:
                     {
                         // Redirect to the SAS URI to blob storage
                        
-                        return (ActionResult)new RedirectResult(inputBlob.GenerateSASURI());
+                        return (ActionResult)new RedirectResult(inputBlob.GenerateSasUri(new BlobSasBuilder()).ToString());
```

Comment:
We would need to set the elements that we want to use in the BlobSasBuilder. @tg-msft is there a sample somewhere that shows an example of this?

Reply:
```suggestion
                        BlobSasBuilder blobSasBuilder = new BlobSasBuilder()
                        {
                            BlobContainerName = inputBlob.BlobContainerName,
                            BlobName = inputBlob.Name,
                            Resource = "b"
                        };
                        return (ActionResult)new RedirectResult(inputBlob.GenerateSasUri(blobSasBuilder).ToString());
```

Reply:
Could we update like this?

Reply:
I have restored the usage in the extension method here, and the latest method of getting sasUri has been updated in the CloudBlockBlobExtensions.cs class